packages/google-pay-integration/e2e/template.ts

```diff
@@ -1,20 +1,22 @@
-import { test, PaymentStepAsGuestPreset } from '../';
+import { test, PaymentStepAsGuestPreset } from '@bigcommerce/checkout/payment-integration-test-framework';
```

Comment:
I moved the file there and commented out all code because `ts` forbids using `@bigcommerce/checkout/payment-integration-test-framework` within `payment-integration-test-framework`, and `eslint` also complains a lot.